packages/flutter_goldens/lib/flutter_goldens.dart

```diff
@@ -521,14 +520,11 @@ class FlutterSkippingFileComparator extends FlutterGoldenFileComparator {
   /// Decides, based on the current environment, if this comparator should be
   /// used.
   ///
-  /// If we are in a CI environment, LUCI or Cirrus, but are not using the other
+  /// If we are in a CI environment, i.e. LUCI, but are not using the other
   /// comparators, we skip. Otherwise we would fallback to the local comparator,
   /// for which failures cannot be resolved in a CI environment.
   static bool isForEnvironment(Platform platform) {
-    return platform.environment.containsKey('SWARMING_TASK_ID')
-        // Some builds are still being run on Cirrus, we should skip these.
-        ||
-        platform.environment.containsKey('CIRRUS_CI');
```

Comment:
This one we might want to keep. This covers _other_ repos that might be running our tests on Cirrus. I think a dart or skia repo might still be using Cirrus, not sure. Happy to remove it if everyone is off Cirrus finally.

The use case here would be a dart repo is running framework tests using Cirrus, but in dart Cirrus CI they cannot affect goldens, so the skipping comparator should be chosen. That way the dart repo does not inadvertently get blocked.

Reply:
Ya know, let's see if we can just remove it. If we can, it's such a nice cleanup.

We'll hear back pretty quick if it breaks someone's CI. We have not been using Cirrus for a really long time, and I have not heard of it being used anymore either.